tools/hmgm/hmgm_ner.xml

```diff
@@ -3,20 +3,20 @@
   <command detect_errors="exit_code">
   	$__tool_directory__/../amp_json_schema/ner_to_iiif.py $input_ner $original_iiif '$context_json';
   	$__tool_directory__/hmgm_main.py NER $__root_dir__ $original_iiif $corrected_iiif $task_info '$context_json';
-    $__tool_directory__/../amp_json_schema/iiif_to_ner.py $corrected_iiif $input_ner $amp_entity_extraction
-</command>                                    
+    $__tool_directory__/../amp_json_schema/iiif_to_ner.py $corrected_iiif $input_ner $output_ner
```

Comment:
@yingfeng-iu for security reasons it would be better to single-quote all text and all data parameters. 

e.g. `'$__tool_directory__/../amp_json_schema/iiif_to_ner.py' '$corrected_iiif' '$input_ner' '$output_ner'`